pyccel/ast/cwrapper.py

```diff
@@ -0,0 +1,51 @@
+from .basic     import Basic
+from .datatypes import DataType
+from .core import FunctionCall, FunctionDef
+__all__ = (
+#
+# --------- CLASSES -----------
+#
+    'PyccelPyObject',
+    'PyArg_ParseTupleNode',
+    'PyBuildValueNode'
+)
+
+class PyArg_ParseTupleNode(Basic):
+
+    def __new__(cls, python_func_args, type_keys='', parse_args=None):
+        return Basic.__new__(cls, python_func_args, type_keys, parse_args)        

```

Comment:
Using `__new__` and `self._args` is still found in lots of places in our code, however we dig into this question a few weeks ago and realised there is no reason to do it this way. We should therefore prefer the method used in the `Variable` class, where everything is a mutable object and is stored in a variable which has a descriptive name